docs/source/about/installation.rst

```diff
@@ -116,12 +116,25 @@ Flatpak Package
 
          flatpak install --user sunshine.flatpak
 
+   Additional installation (required)
+      .. code-block:: bash
+
+         flatpak run --command=additional-install.sh dev.lizardbyte.sunshine
+
 Start:
    X11 and NVFBC capture (X11 Only)
+      Direct start
+
       .. code-block:: bash
 
          flatpak run dev.lizardbyte.sunshine
 
+      Sunshine Service
+
+      .. code-block:: bash
+
+         systemctl --user start sunshine
+
```

Comment:
This is already covered in the `usage` page. I probably wouldn't add it here since we don't have it for the other linux versions.